spec/features/merchants/show_spec.rb

```diff
@@ -0,0 +1,68 @@
+# spec/features/merchants/dashboard_spec
+
+require 'rails_helper'
+
+RSpec.describe 'Merchants dashboard show page' do 
+  describe 'display' do
+    before :each do 
```

Comment:
You can keep writing tests this way for now; it shouldn't interfere with my setting up the test-database